include/vcpkg/base/format.h

```diff
@@ -25,7 +25,7 @@ namespace fmt
     template<>
     struct formatter<vcpkg::LineInfo, char>
     {
-        constexpr auto parse(format_parse_context& ctx) const -> decltype(ctx.begin())
+        static constexpr auto parse(format_parse_context& ctx) -> decltype(ctx.begin())
```

Comment:
As I indicated in the other PR, this is following the formatter protocol, so this change damages correctness. https://fmt.dev/latest/api.html#formatting-user-defined-types

(fmt wants to call with `a.b` syntax not `a::b` syntax)

Adding const would be OK.